src/module_wrap.cc

```diff
@@ -98,99 +98,87 @@ ModuleWrap* ModuleWrap::GetFromID(Environment* env, uint32_t id) {
   return module_wrap_it->second;
 }
 
-// new ModuleWrap(source, url)
-// new ModuleWrap(source, url, context?, lineOffset, columnOffset)
-// new ModuleWrap(syntheticExecutionFunction, export_names, url)
+// new ModuleWrap(url, context, source, lineOffset, columnOffset)
+// new ModuleWrap(url, context, exportNames, syntheticExecutionFunction)
```

Comment:
Since `context` is optional in most uses it would be nicer if we could keep it an optional argument, eg `(url, exportNames, syntheticExecutionFunction, context)`, but I'm ok with this too so no pressure.